vhdbuilder/packer/windows/windows_settings.json

```diff
@@ -39,14 +39,28 @@
       "os_disk_size": "35",
       "base_image_sku": "23h2-datacenter-core",
       "windows_image_name": "windows-23H2",
-      "base_image_version": "25398.1425.250210",
+      "base_image_version": "",
```

Comment:
This change seems odd - I might have done it by accident, but we should revert it.